### PR TITLE
Add APISIX implementation

### PR DESCRIPTION
**Pull request by tao12345666333**
Signed-off-by: Jintao Zhang <zhangjintao9020@gmail.com>

**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: site-src/implementations.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ Implementors of Gateway API are encouraged to update this document with status i
 - [Istio][6] (alpha)
 - [Kong][7] (work in progress)
 - [Traefik][8] (alpha)
+- [Apache APISIX][9] (work in progress)
```
Comment (robscott) [inline site-src/implementations.md:17]: Tiny nit: I think the list of implementations has been alphabetical so far which would mean your implementation should appear first in the list.

```suggestion
- [Apache APISIX][9] (work in progress)
```

**Review by robscott — COMMENTED** [review]
Thanks for the work on this! LGTM other than one tiny nit.

**Comment by robscott** [comment]
Looks like a flake with the kind cluster.

/retest

File: site-src/implementations.md [hunk 2]
```diff
@@ -118,3 +120,13 @@ Traefik is currently working on implementing TCP, status updates and documentati
 
 [traefik]:https://traefik.io
 [traefik-1]:https://doc.traefik.io/traefik/routing/providers/kubernetes-gateway/
+
+
+### APISIX
+
+[Apache APISIX][apisix] is a dynamic, real-time, high-performance API Gateway. APISIX provides rich traffic management features such as load balancing, dynamic upstream, canary release, circuit breaking, authentication, observability, and more.
+
+APISIX is moving forward to implement the Gateway API `v1alpha2` version of the specification for it's [Apache APISIX Ingress Controller][apisix-1] and status updates and documentation will be provided here as the work progresses.
```
Comment (jpeach) [inline site-src/implementations.md:129]:
```suggestion
APISIX is moving forward to implement the Gateway API `v1alpha2` version of the specification for its [Apache APISIX Ingress Controller][apisix-1], and status updates and documentation will be provided here as the work progresses.
```

File: site-src/implementations.md (hunk 1 above)
Comment (tao12345666333) [reply site-src/implementations.md:17]: Thanks！Let me update it.

Comment (tao12345666333) [reply site-src/implementations.md]: Addressed, Thanks!

**Comment by robscott** [comment]
Thanks @tao12345666333!

/approve

**Pull request merged**